src/js/common/components/form-elements/ExpandingGroup.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+
+export default function ExpandingGroup({ children, open, additionalClass }) {
```

Comment:
I like using the stateless function components when possible, since it's simpler syntax.